input/fsh/moped-operation-verlegen.fsh

```diff
@@ -21,7 +21,7 @@ Die Operation wird vom Akteur Krankenhaus (KH) aufgerufen. Die $verlegen Operati
   * *MOPEDTransferEncounter.serviceProvider* setzt eine Referenz auf die MOPEDOrganizationAbteilung mit dem jeweiligen *funktionscode* bzw. *funktionssubcode* lt. Operation-Parameter.
   * *MOPEDTransferEncounter.Neugeborenes* wird lt. LKF-Regeln berechnet, anhand des *MOPEDEncounter.subject.birthdate* aus dem Encounter aus Schritt 1 (für Berechnugns-Details siehe Hinweis 1).
   * *MOPEDTransferEncounter.Altersgruppe* wird lt. LKF-Regeln berechnet, anhand des *MOPEDEncounter.subject.birthdate* aus dem Encounter aus Schritt 1 (für Berechnugns-Details siehe Hinweis 2).
-  * *MOPEDTransferEncounter.PhysischeAnwesenheit* wird lt. Operation-Parameter befüllt.
+  * *MOPEDTransferEncounter.anwesenheitsart* wird lt. Operation-Parameter befüllt.
```

Comment:
Der Pfad wäre `MOPEDTransferEncounter.subjectStatus.coding.code` bzw. kann im gleichen Zug durch Nachsehen in dem ValueSet auch gleich automatisiert `MOPEDTransferEncounter.subjectStatus.coding.display` befüllt werden